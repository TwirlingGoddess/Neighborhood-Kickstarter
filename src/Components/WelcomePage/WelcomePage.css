.user-link-section {
  display: flex;
  width: 100%;
  height: 100vh;
  justify-content: center;
}

.user-link {
  box-sizing: border-box;
  appearance: none;
  background-color: transparent;
  border-radius: 0.6em;
  cursor: pointer;
  align-self: center;
  font-size: 1rem;
  font-weight: 400;
  line-height: 1;
  margin: 20px;
  padding: 1em 1em;
  text-decoration: none;
  text-align: center;
  text-transform: uppercase;
  font-family: 'Montserrat', sans-serif;
  font-weight: 700;
  border-color: #3498db;
  color: #fff;
  box-shadow: 0 0 40px 40px #3498db inset, 0 0 0 0 #3498db;
  transition: all 150ms ease-in-out;
  width: 12rem;
  max-height: 4rem;
}

.user-link:hover {
  color: #fff;
  outline: 0;
  box-shadow: 0 0 10px 0 #3498db inset, 0 0 10px 4px #3498db;
}

.signin {
  height: 5rem;
}