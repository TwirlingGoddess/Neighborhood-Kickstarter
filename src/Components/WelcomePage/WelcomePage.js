import React, { Component } from 'react';

class WelcomePage extends Component {
  constructor() {
    super()
    this.state = ''
  }
}

export default WelcomePage