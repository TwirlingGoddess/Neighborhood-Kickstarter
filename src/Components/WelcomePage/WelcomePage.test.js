import React from 'react';
import WelcomePage from './WelcomePage';
import { shallow } from 'enzyme';

describe('WelcomePage', () => {
  let wrapper;
  it('should match the snapshot', () => {
    wrapper = shallow(<WelcomePage />);
    expect(wrapper).toMatchSnapshot();
  });
});  