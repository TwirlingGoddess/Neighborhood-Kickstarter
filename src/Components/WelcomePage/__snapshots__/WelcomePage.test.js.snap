// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomePage should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WelcomePage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="welcome-heading-section"
        >
          <h1
            className="welcome-heading"
          >
            Welome to
          </h1>
          <h1
            className="welcome-heading"
          >
            NeighborHub
          </h1>
        </div>,
        <div
          className="welcome-button-section"
        >
          <NavLink
            activeClassName="active"
            aria-current="page"
            className="user-link"
            to="/Landing"
          >
            View Projects
          </NavLink>
          <NavLink
            activeClassName="active"
            aria-current="page"
            className="user-link"
            to="/SignIn"
          >
            Sign In
          </NavLink>
          <button
            className="user-link"
            onClick={[Function]}
            to="/SignIn"
          >
            Create Account
          </button>
        </div>,
      ],
      "className": "user-link-section",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1
              className="welcome-heading"
            >
              Welome to
            </h1>,
            <h1
              className="welcome-heading"
            >
              NeighborHub
            </h1>,
          ],
          "className": "welcome-heading-section",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Welome to",
              "className": "welcome-heading",
            },
            "ref": null,
            "rendered": "Welome to",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "NeighborHub",
              "className": "welcome-heading",
            },
            "ref": null,
            "rendered": "NeighborHub",
            "type": "h1",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <NavLink
              activeClassName="active"
              aria-current="page"
              className="user-link"
              to="/Landing"
            >
              View Projects
            </NavLink>,
            <NavLink
              activeClassName="active"
              aria-current="page"
              className="user-link"
              to="/SignIn"
            >
              Sign In
            </NavLink>,
            <button
              className="user-link"
              onClick={[Function]}
              to="/SignIn"
            >
              Create Account
            </button>,
          ],
          "className": "welcome-button-section",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": "View Projects",
              "className": "user-link",
              "to": "/Landing",
            },
            "ref": null,
            "rendered": "View Projects",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": "Sign In",
              "className": "user-link",
              "to": "/SignIn",
            },
            "ref": null,
            "rendered": "Sign In",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Create Account",
              "className": "user-link",
              "onClick": [Function],
              "to": "/SignIn",
            },
            "ref": null,
            "rendered": "Create Account",
            "type": "button",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="welcome-heading-section"
          >
            <h1
              className="welcome-heading"
            >
              Welome to
            </h1>
            <h1
              className="welcome-heading"
            >
              NeighborHub
            </h1>
          </div>,
          <div
            className="welcome-button-section"
          >
            <NavLink
              activeClassName="active"
              aria-current="page"
              className="user-link"
              to="/Landing"
            >
              View Projects
            </NavLink>
            <NavLink
              activeClassName="active"
              aria-current="page"
              className="user-link"
              to="/SignIn"
            >
              Sign In
            </NavLink>
            <button
              className="user-link"
              onClick={[Function]}
              to="/SignIn"
            >
              Create Account
            </button>
          </div>,
        ],
        "className": "user-link-section",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h1
                className="welcome-heading"
              >
                Welome to
              </h1>,
              <h1
                className="welcome-heading"
              >
                NeighborHub
              </h1>,
            ],
            "className": "welcome-heading-section",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Welome to",
                "className": "welcome-heading",
              },
              "ref": null,
              "rendered": "Welome to",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "NeighborHub",
                "className": "welcome-heading",
              },
              "ref": null,
              "rendered": "NeighborHub",
              "type": "h1",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <NavLink
                activeClassName="active"
                aria-current="page"
                className="user-link"
                to="/Landing"
              >
                View Projects
              </NavLink>,
              <NavLink
                activeClassName="active"
                aria-current="page"
                className="user-link"
                to="/SignIn"
              >
                Sign In
              </NavLink>,
              <button
                className="user-link"
                onClick={[Function]}
                to="/SignIn"
              >
                Create Account
              </button>,
            ],
            "className": "welcome-button-section",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": "View Projects",
                "className": "user-link",
                "to": "/Landing",
              },
              "ref": null,
              "rendered": "View Projects",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": "Sign In",
                "className": "user-link",
                "to": "/SignIn",
              },
              "ref": null,
              "rendered": "Sign In",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Create Account",
                "className": "user-link",
                "onClick": [Function],
                "to": "/SignIn",
              },
              "ref": null,
              "rendered": "Create Account",
              "type": "button",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
